src/pages/index.fr-fr.js

```diff
@@ -0,0 +1,198 @@
+les opérations réseau. La combinaison de ces informations forme une fondation
+extraordinaire pour construire de nouvelles approches pour sécuriser les
+systèmes d'information. Alors même que le filtrage des appels système, des flux
+réseaux, et le suivi du contexte d'exécution des processus sont typiquement
+gérés par des solutions distinctes, eBPF offre à la fois vision et contrôle sur
+tous les composants, ouvrant ainsi la voie à un système de sécurisation qui
+porte sur un domaine plus large, et propose un contrôle plus poussé.
+`;
+
+const networkingText = `
+La programmabilité et la performance d'eBPF en font une solution idéale pour le
+traitement de paquets pour tous types d'applications réseau. Programmer avec
+eBPF permet d'analyser et d'implémenter de nouveaux protocoles, et d'appliquer
+une logique de routage des paquets pour répondre à des besoins en évolution
+constante, sans jamais quitter le contexte du noyau Linux. Le compilateur JIT
+(« Just-In-Time ») permet d'atteindre, à l'exécution des programmes, des
+performances similaires à celles du code compilé nativement dans le noyau.
+`;
+
+const monitoringText = `
+Au lieu de se baser sur des jauges et des compteurs statiques exposés par le
+système d'exploitation, eBPF collecte et agrège dans le noyau des métriques
+personnalisées, et rend possible la génération d'évènements d'observation basés
+sur un large éventail de sources disponibles. Seules les données utiles sont
+relevées, et les représentations telles que les histogrammes sont générées à la
+source de l'évènement plutôt qu'à la réception d'un flot d'échantillons
+exportés du noyau. eBPF offre ainsi une visibilité du système avec une portée
+accrue, tout en réduisant significativement le cout associé à la collecte de
+données.
+`;
+
+const Buttons = () => (
+  <h1 className="main-buttons">
+    <Link to="/what-is-ebpf" className="main-button">
+      Comprendre eBPF
+    </Link>
+    <Link to="/projects" className="main-button">
+      Projets
+    </Link>
+  </h1>
+);
+
+const Intro = () => (
+  <div className="intro">
+    <p>
+      Le noyau Linux a toujours été l'endroit idéal pour implémenter les
+      aspects inspection/supervision, réseau, et sécurité du système.
+      Malheureusement, la nécessité de changer le code du noyau ou de charger
+      des modules rend la tâche contraignante, et revient souvent à empiler les
+      couches d'abstraction. eBPF est une technologie révolutionnaire
+      permettant d'exécuter des programmes au sein du noyau Linux, dans un
+      environnement confiné, sans changer le code source du noyau ni charger de
+      modules.
+    </p>
+    <p>
+      Grâce à cette capacité à programmer le noyau, les composants logiciels
+      peuvent s'appuyer sur le système existant. Ils viennent le perfectionner
+      et l'enrichir en fonctionnalités, en évitant de rajouter des couches
+      d'abstraction, mais sans compromettre les performances ou la sureté du
```

Comment:
I think that "efficiency" is better translated as "efficacité" instead of "performance" in this context.

Reply:
_Entre les deux mon cœur balance._ “Performance” sounded slightly better in my mind, but I can change this one if you believe the opposite.

Reply:
No strong opinion but it seems to me that "efficacité" is closer to the original term/meaning.